lib/swagger-module.ts

```diff
@@ -65,19 +65,21 @@ export class SwaggerModule {
     httpServer: any,
     document: OpenAPIObject
   ) {
-    httpServer.register(
-      loadPackage('fastify-swagger', 'SwaggerModule', () =>
-        require('fastify-swagger')
-      ),
-      {
-        swagger: document,
-        exposeRoute: true,
-        routePrefix: path,
-        mode: 'static',
-        specification: {
-          document
+    httpServer.register(async (httpServer: any) => {
```

Comment:
what happens here?

Reply:
Apparently fastify plugin works differently. Per its [documentation](https://www.fastify.io/docs/latest/Plugins/), this adds a different scope so the internal call
`fastify.decorate('swagger', swagger)` does not collide when we try to make multiple swagger docs